docker/py/Dockerfile

```diff
@@ -59,7 +59,7 @@ RUN conda install gxx_linux-64 && \
 ENV RENKU_DISABLE_VERSION_CHECK 1
 
 ARG RENKU_PIP_SPEC="renku"
-RUN pipx install ${RENKU_PIP_SPEC} --pip-args="--pre --no-cache" && \
+RUN pipx install ${RENKU_PIP_SPEC} --pip-args="--no-cache" && \
```

Comment:
we might have to call this `${RENKU_PIP_SPEC}==$RENKU_VERSION` or something like this? i could be wrong but between the template install with .gitlab-ci.yml and the image install i think something isn't lining up between the name of the args and the renku version

Reply:
`RENKU_PIP_SPEC` is supposed to be something like `renku==0.10.4` or `renku>=0.10.0` or similar. `RENKU_VERSION` would be just `0.10.4`.

Reply:
yeah but i think you took out the line that adds RENKU_VERSION to RENKU_PIP_SPEC

Reply:
no - that was done in the github action before. The Dockerfile only had `RENKU_PIP_SPEC`. See https://github.com/SwissDataScienceCenter/renkulab-docker/pull/112/files#diff-7453a2391fb8e2a749f878b0fc2126acL36

We could remove this completely, but then there is no way to install a different default version in there without changing the Dockerfile so I prefer to leave it in.

Reply:
i mean you didn't put anything in here to pass in RENKU_PIP_SPEC as an arg: https://github.com/SwissDataScienceCenter/renku-project-template/pull/86/files

Reply:
That's right - the configuration there doesn't interact with this Dockerfile, it just uses the resulting image.